azavea/district-builder-dtl-pa
# Change favicon to DTL logo

## Overview

Changes the favicon to match the DTL logo.

### Checklist


### Demo

<img width="155" alt="screen shot 2018-08-24 at 5 49 51 pm" src="https://user-images.githubusercontent.com/447977/44588200-215e6e00-a7c6-11e8-982e-67954f4be714.png">

### Notes

- Favicons are notorious for being aggressively cached by browsers; rather than force everyone at C70 to clear their browser cache, I put in a poor-man's cache-busting query parameter. We'll have to manually keep changing it if we need to make further updates to this.

## Testing Instructions

 * Load the homepage, confirm the favicon matches the one in the task.
 * Log in and do the same elsewhere in the app.

Closes #159976095
